packages/modules/sma_sunny_boy/device.py

```diff
@@ -174,6 +174,7 @@ def create_modbus_inverter(address: str) -> inverter.SmaSunnyBoyInverter:
                     total_power -= bat_state.power
             else:
                 total_power = 0
+            print("WR 1 nach Korrektur: {}".format(state))
         for inv in itertools.chain(inverters_additional):
             state = inv.read()[0]
```

Comment:
Ich würde hier auch noch den state aller weiteren WR loggen.

Reply:
Die Werte werden nach dem Auslesen in inverter.py geloggt.